guides/kfp_tekton_install.md

```diff
@@ -35,18 +35,24 @@ To install the standalone Kubeflow Pipelines with Tekton, run the following step
 
 1. Install [Tekton v0.21.0](https://github.com/tektoncd/pipeline/releases/tag/v0.21.0)
 
-2. Install Kubeflow Pipelines with Tekton backend (`kfp-tekton`) `v0.6.0` [custom resource definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)(CRDs).
+2. Enable custom task for pipeline loops and other feature flags for kfp-tekton
+   ```shell
+    kubectl patch cm feature-flags -n tekton-pipelines \
+        -p '{"data":{"disable-home-env-overwrite":"true","disable-working-directory-overwrite":"true", "enable-custom-tasks": "true"}}'
+   ```
+
+3. Install Kubeflow Pipelines with Tekton backend (`kfp-tekton`) `v0.7.0` [custom resource definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)(CRDs).
    > Note: You can ignore the error `no matches for kind "Application" in version "app.k8s.io/v1beta1"` since it's a warning saying `application` CRD is not yet ready.
     ```shell
     kubectl apply --selector kubeflow/crd-install=true -f install/v0.6.0/kfp-tekton.yaml
```

Comment:
should this be `0.7.0`:
`kubectl apply --selector kubeflow/crd-install=true -f install/v0.7.0/kfp-tekton.yaml` instead of
`kubectl apply --selector kubeflow/crd-install=true -f install/v0.6.0/kfp-tekton.yaml` ?